opm/input/eclipse/Schedule/ScheduleState.cpp

```diff
@@ -388,6 +388,10 @@ void ScheduleState::update_events(Events events) {
     this->m_events = events;
 }
 
+void ScheduleState::merge_events(const Events& events) {
```

Comment:
just to point out the obvious: there are mutable accessors so these are not strictly necessary.